package.json

```diff
@@ -19,7 +19,22 @@
   ],
   "main": "./out/extension",
   "contributes": {
-    "commands": []
+    "commands": [],
```

Comment:
wait a second, I should remove this line, since there are no commands contributed